doc/dash/dash-sonic-hld.md

```diff
@@ -695,11 +707,18 @@ DASH_TUNNEL_TABLE:{{tunnel_name}}
 key                      = DASH_TUNNEL_TABLE:tunnel_name; tunnel name used for referencing in mapping table
 ; field                  = value
 endpoints                = list of addresses for ecmp tunnel
-encap_type               = vxlan or nvgre
-vni                      = vni value for encap
+encap_type               = vxlan or nvgre, create only attribute
```

Comment:
For the `encap_type` can we add an explicit definishion of what is expected return packet?

Reply:
added a statement